instance_disks.go

```diff
@@ -61,15 +61,13 @@ type InstanceDiskCreateOptions struct {
 	Filesystem      string            `json:"filesystem,omitempty"`
 	AuthorizedKeys  []string          `json:"authorized_keys,omitempty"`
 	AuthorizedUsers []string          `json:"authorized_users,omitempty"`
-	ReadOnly        bool              `json:"read_only,omitempty"`
 	StackscriptID   int               `json:"stackscript_id,omitempty"`
 	StackscriptData map[string]string `json:"stackscript_data,omitempty"`
 }
 
 // InstanceDiskUpdateOptions are InstanceDisk settings that can be used in updates
 type InstanceDiskUpdateOptions struct {
```

Comment:
It seems `filesystem` and `size` can also be updated: https://www.linode.com/docs/api/linode-instances/#disk-update. Probably update them as well?

Reply:
Unfortunately, the doc is incorrect... I also have [a PR for fixing that doc](https://github.com/linode/linode-api-docs/pull/924).

Reply:
Thank you for taking care of everything!